### ISSUE TITLE
[Fix site]: WSJ gets a captcha since January; no links

### ISSUE THREAD

**Issue by jeremybmerrill** [palewire/news-homepages#486]
## Screenshot

![wsj](https://github.com/user-attachments/assets/da07212b-22a7-4e69-8c80-9f6b27a1ae14)

Screenshot via https://palewi.re/docs/news-homepages/sites/wsj.html. AFAIK this has been going since 2024-01-16 16:11:00 (based on the last non-empty links JSON.) I wonder if this is something that could be rectifying (and backfilled) by scraping Internet Archive screenshots, or by asking WSJ to allowlist you. I also wonder if the captcha is part of WSJ's anti-AI-training-data-scraping efforts.

Have you circumvented captchas in other sites? 

> A solution is often found by adding JavaScript or CSS via a site-specific include, [as covered in our documentation](https://palewi.re/docs/news-homepages/adding.html#hide-ads-and-popups).

retaining this boilerplate from the template :)

**Comment by palewire** [comment]
I'm aware of this bug, but I haven't even begun to think about how to fix it. I'm open to patches, and connex to people at WSJ to consult.

**Comment by jeremybmerrill** [comment]
Fair enough. Curious why you're gathering the pages yourself, rather than getting them from the Internet Archive (which appears to have circumvented the WSJ's limitations.)

**Comment by palewire** [comment]
It's probably a longer story than anyone wants to hear. I launched the site in [2012](https://www.wsj.com/articles/BL-DGB-24386) independent of archive.org as a self-hosted service funded by a Kickstarter campaign. At that time the Wayback Machine was not archiving the homepages of major sites with much frequency. There have been several evolutions since, with the currently one hosting assets for free with IA's generous "collections" system. 

It would be possible to re-engineer the site to act as a supplement to Wayback's page captures. And perhaps I should move towards such a system. In the 12 years since I started, IA has ramped up its capturing rate for big sites. Though that's not always the case for lower trafficked sites.

**Comment by jeremybmerrill** [comment]
No, that's very useful context, thank you! It feels like the idea I'd be most open to implementing: scraping IA, would be a larger re-engineering of the project that I can't really take on.

**Comment by jeremybmerrill** [comment]
Just flagging the Reuters has the same issue as of 2023-12-04 03:59:00
